relayer-cli/src/devnetRelayExample.ts

```diff
@@ -47,7 +46,8 @@ async function initialize(chain_id: number): Promise<number> {
   fs.writeFileSync(lock_file_name, process.pid.toString(), { encoding: "utf8" });
 
   const state_file_name = "./state/" + chain_id + ".json";
-  if (!fs.existsSync(state_file_name)) {
+  const state_file_name_fs_check = "./src/state/" + chain_id + ".json";
+  if (!fs.existsSync(state_file_name_fs_check)) {
     // No state file so initialize starting now
```

Comment:
Use an env variable to set the path, like `$STATE_DIR` and set a default value in `.env.example` outside the src/ folder